src/software/world/ball_state.h

```diff
@@ -56,12 +49,9 @@ class BallState final
      * @return True if the other ball state is not equal to this ball state, and false
      * otherwise
      */
-    bool operator!=(const BallState &other) const;
+    bool operator!=(const BallState& other) const;
 
-   private:
+   protected:
```

Comment:
What was the reasoning behind making these `protected`?

Reply:
Thought I needed direct access in BallStateWithTimestamp, but I do not. Made private.